qiskit_experiments/curve_analysis/base_curve_analysis.py

```diff
@@ -338,7 +337,7 @@ def _evaluate_quality(
         Returns:
             String that represents fit result quality. Usually "good" or "bad".
         """
-        if fit_data.reduced_chisq < 3.0:
+        if 0 < fit_data.reduced_chisq < 3.0:
```

Comment:
This `chisq > 0` condition has been added to `quality=good` because of edge cases where zero `chisq` is a sign that something is wrong (in this case, running RB with only one sample per length)
![Pasted image 20230728110246](https://github.com/Qiskit-Extensions/qiskit-experiments/assets/3870315/ec0f4379-d341-49aa-8ac7-fe9c7e3c42e5)

Reply:
Thank you for the explanation. This change itself makes sense to me. (I'm not sure we can really determine if a fit is good or bad based on a single value, though, but I know it's another issue.)